### PR TITLE
feat: add autoSyncForm markup

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [ ] Not that difficult
- [ ] Approved feature branch merge to master

### Description
image
SSIA

### CODE REVIEW

File: apps/web/src/services/asset-inventory/components/ServiceAccountAutoSyncForm.vue [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+});
+
+const emit = defineEmits<{(e:'update:isValid', isValid: boolean): void;
+    (e:'change', formData: any): void;
+}>();
+
+const state = reactive({
+    timezone: computed<string>(() => store.state.user.timezone),
+    scheduleHelpText: computed(() => i18n.t('INVENTORY.SERVICE_ACCOUNT.CREATE.TIMEZONE', { timezone: state.timezone })),
+    additionalOptions: {},
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/components/ServiceAccountAutoSyncForm.vue:43]: nitpick: please specify the type!

File: apps/web/src/services/asset-inventory/components/ServiceAccountDeleteModal.vue [hunk 2]
```diff
@@ -93,16 +93,12 @@ const handleConfirmDelete = async () => {
                         :visible.sync="state.proxyVisible"
                         :header-title="$t('INVENTORY.SERVICE_ACCOUNT.DELETE_CHECK_MODAL.TITLE')"
                         theme-color="alert"
-                        :hide-header-close-button="true"
-                        :hide-footer-confirm-button="true"
+                        size="sm"
+                        :hide-footer-close-button="true"
+                        @confirm="() => state.proxyVisible = false"
```
Comment (piggggggggy) [inline apps/web/src/services/asset-inventory/components/ServiceAccountDeleteModal.vue:98]: Suggestion: separating event function is better! like `handleConfirm~`

**Review by piggggggggy — APPROVED** [review]
LGTM~

**Review by yuda110 — APPROVED** [review]
LGTM!!!

**Pull request merged**